include/deal.II/grid/manifold_lib.h

```diff
@@ -796,7 +796,8 @@ class TransfiniteInterpolationManifold : public Manifold<dim,spacedim>
    */
   DerivativeForm<1,dim,spacedim>
   push_forward_gradient(const typename Triangulation<dim,spacedim>::cell_iterator &cell,
-                        const Point<dim> &chart_point) const;
+                        const Point<dim>      &chart_point,
+                        const Point<spacedim> &interpolated_chart_point) const;
```

Comment:
It took me a while to understand that you are allowed to do this because the class is not derived from `ChartManifold`. Can you copy the comment from the previous two functions here as well?

Reply:
Yes indeed, this is a private function and different from `ChartManifold`. I added the respective comment (and fixed a slight inaccuracy in one of the others).